assets/sass/_vars.scss

```diff
@@ -39,6 +39,16 @@ $accentBorderWidth: 4px;
 // (and also referenced in bordered grid)
 $contentBoxHorizontalPadding: 20px;
 
+/* =========================================================================
+   Global Variables: Font Stacks
+   ========================================================================= */
+
+$font-stacks: (
```

Comment:
Maybe add a comment here to remind us that changes here should be reflected in `assets/js/head.js` too?